src/java/org/apache/cassandra/gms/Gossiper.java

```diff
@@ -123,6 +123,9 @@ public static class Props
         SILENT_SHUTDOWN_STATES.add(VersionedValue.STATUS_BOOTSTRAPPING);
         SILENT_SHUTDOWN_STATES.add(VersionedValue.STATUS_BOOTSTRAPPING_REPLACE);
     }
+
+    static final List<String> BOOTSTRAPPING_STATES = Arrays.asList(VersionedValue.STATUS_BOOTSTRAPPING,
```

Comment:
This duplicates `org.apache.cassandra.gms.VersionedValue#BOOTSTRAPPING_STATUS`

Reply:
👍  That was added in 5.0. I can add it to this patch to minimize the diff between versions